doc/user/content/integrations/_index.md

```diff
@@ -137,20 +137,13 @@ Materialize integrates with dbt through the [`dbt-materialize`](https://github.c
 | dbt Core  | {{< supportLevel beta >}}        | See the [dbt documentation](https://docs.getdbt.com/reference/warehouse-profiles/materialize-profile) for more details, and the [dbt + Materialize guide](/integrations/dbt/) for a step-by-step breakdown of the integration. | [](#notify) |
 | dbt Cloud | {{< supportLevel in-progress >}} | Not supported yet. We are working with the dbt community to bring native Materialize support to dbt Cloud soon.                                                                                                                | [](#notify) |
 
-### Terraform
+### Infrastructure as Code
 
-Materialize maintains a
```

Comment:
I think all of this can be safely removed since it is covered in the terraform guide. Debating if the note on module support should be moved as well.